.github/workflows/pymeasure_CI.yml

```diff
@@ -154,3 +154,43 @@ jobs:
         run: |
           echo "::add-matcher::.github/pytest.json"
           xvfb-run -a pytest
+
+  test_python312:
+    name: Python ${{ matrix.python-version }}, ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+        python-version: ["3.12"]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install pymeasure requirements
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: .github/pymeasure_python312.yml
+          create-args: python=${{ matrix.python-version }}
+          cache-environment-key: py${{ matrix.python-version }}-${{ matrix.os }}-mamba-${{ env.CACHE_NUMBER }}-${{ hashFiles('environment.yml') }}
+          cache-downloads: false
```

Comment:
This is the only difference right, grabbing a different environmenet file?

As this is only a temporary solution, I'm fine with this duplicate job. But maybe if we want to reduce duplicates, it might also be possible to combine it with the main test job using some conditions (e.g., `if: ${{ matrix.python-version }} == "3.12"` and `if: ${{ matrix.python-version }} != "3.12"`).

Reply:
Good point, I'll try it.

Reply:
It works!